pkg/controller/control.go

```diff
@@ -613,25 +631,28 @@ func (c *Controller) Start(addresses ...string) error {
 
 	c.reset()
 
-	var fatalErr error
 	availableBackends := map[string]types.Backend{}
+	backendSizes := map[int64]struct{}{}
```

Comment:
Why not just use list?

Reply:
Because I just want to check how many different sizes in the backens. Using list needs to iterate the values in the list for every new size.